packages/gatsby-telemetry/src/event-storage.ts

```diff
@@ -114,15 +114,14 @@ module.exports = class EventStorage {
     }
   }
 
-  getConfig(key: string): string {
+  getConfig(key: string): string | Record<string, any> {
```

Comment:
You could make this `Record<string, unknown>`

Reply:
it was `Object` when I committed this 😱

Reply:
Updated